src/prime_rl/orchestrator/orchestrator.py

```diff
@@ -208,6 +210,9 @@ async def orchestrate(config: OrchestratorConfig):
     max_steps = config.max_steps or int(1e9)
     logger.info(f"Starting orchestrator loop (max_steps={max_steps or 'infinite'})")
     last_eval_step = -1
+    if resumed_from_checkpoint and config.eval and config.eval.skip_eval_on_resume:
```

Comment:
@cursoragent does this make sense? could we not just set last_eval_step when we enter the resume and set resumed_from_checkpoint = True? seems like this adds an unnecessary variable to track

Reply:
@cursoragent try again

Reply:
@cursoragent now the resumed_ckpt_step is unecessary?